tests/test__collection_api.py

```diff
@@ -2065,6 +2065,30 @@ def test__aggregate_sample_errors(self):
             with self.assertRaises(mongomock.OperationFailure):
                 self.db.a.aggregate([{'$sample': case}])
 
+    def test__aggregate_count(self):
+        self.db.a.insert_many([
+            {'_id': 1, 'a': 1},
+            {'_id': 2, 'a': 2},
+            {'_id': 3, 'a': 1}
+        ])
+
+        actual = list(self.db.a.aggregate([
+            {'$match': {'a': 1}},
+            {'$count': 'one_count'}
+        ]))
+        self.assertEqual([{'one_count': 2}], actual)
+
+    def test__aggregate_count_errors(self):
+        self.db.a.insert_many([
+            {'_id': i}
+            for i in range(5)
+        ])
+        # Many cases for '$count' options that should raise an operation failure.
+        cases = (None, 3, {}, '', [])
```

Comment:
please also add `$one_count` and `one.count` as cases that should raise an error (note that you'll need to update the code so that it actually raises the error).

Reply:
Thank you ! I add the code in https://github.com/mongomock/mongomock/pull/479/commits/35568b11e21639c85e917a35c448a7c3fd7199f0